core/types/transaction_marshalling.go

```diff
@@ -30,15 +30,19 @@ type txJSON struct {
 	Type hexutil.Uint64 `json:"type"`
 
 	// Common transaction fields:
-	Nonce    *hexutil.Uint64 `json:"nonce"`
-	GasPrice *hexutil.Big    `json:"gasPrice"`
-	Gas      *hexutil.Uint64 `json:"gas"`
-	Value    *hexutil.Big    `json:"value"`
-	Data     *hexutil.Bytes  `json:"input"`
-	V        *hexutil.Big    `json:"v"`
-	R        *hexutil.Big    `json:"r"`
-	S        *hexutil.Big    `json:"s"`
-	To       *common.Address `json:"to"`
+	Nonce                *hexutil.Uint64 `json:"nonce"`
+	GasPrice             *hexutil.Big    `json:"gasPrice"`
+	FeeCap               *hexutil.Big    `json:"feeCap"`
+	Tip                  *hexutil.Big    `json:"tip"`
+	MaxPriorityFeePerGas *hexutil.Big    `json:"maxPriorityFeePerGas"`
+	MaxFeePerGas         *hexutil.Big    `json:"maxFeePerGas"`
```

Comment:
Any particular reason for the ambiguity from day 0? Seems weird to allow two names for the same thing.

Reply:
Especially since we only use FeeCap and Tip during serialization, so client code needs to implement the "canonical" fields either way. Why bother allowing a non-canonical version?

Reply:
You're right, probably best to stick to one naming scheme. The reason for the ambiguity is a final name hasn't been decided upon for the RPC methods.